### DISCUSSION TITLE
How to add big item to Application sub Menu

### DISCUSSION THREAD

**Discussion by rulai-yinqiang** [harborsiem/WindowsRibbon#32] [Q&A] [answered]
How to add Big items to Application sub Menu like Windows Explorer ?

![image](https://github.com/harborsiem/WindowsRibbon/assets/33405365/00d0bd4b-69ce-4559-b51b-6b03480a1536)

I can only add Small items.
![image](https://github.com/harborsiem/WindowsRibbon/assets/33405365/ded60437-6a8e-4398-a75d-0517ce43895d)

**Comment by harborsiem** [comment, marked as answer]
[See](https://learn.microsoft.com/en-us/windows/win32/windowsribbon/windowsribbon-element-menugroup)
and look to examples 03-MenuDropDown
